composable_kernel/include/tensor_operation/gridwise_gemm_xdlops_v2r3.hpp

```diff
@@ -53,6 +53,63 @@ __global__ void
                                                   c_element_op,
                                                   block_2_ctile_map);
 }
+
+template <typename GridwiseGemm,
+          typename FloatAB,
+          typename FloatC,
+          typename AGridDesc_K0_M_K1,
+          typename BGridDesc_K0_N_K1,
+          typename CGridDesc_M0_N0_M1_N1_M2_M3_M4_N2,
+          typename AElementwiseOperation,
+          typename BElementwiseOperation,
+          typename CElementwiseOperation,
+          typename Block2CTileMap,
+          bool HasMainKBlockLoop>
+__global__ void
+#if CK_USE_LAUNCH_BOUNDS
+    __launch_bounds__(CK_MAX_THREAD_PER_BLOCK, CK_MIN_BLOCK_PER_CU)
```

Comment:
I think you can move this function into DeviceOp for your conv3d

Reply:
Makes sense. Will do.